cloudwatch_logs_common/src/utils/log_file_manager.cpp

```diff
@@ -34,13 +39,18 @@ FileObject<LogCollection> LogFileManager::readBatch(
   /* We must sort the log data chronologically because it is not guaranteed
```

Comment:
This comment isn't relevant here any longer. We should document where relevant in the code and in the method header as well.